diffblue.yml

```diff
@@ -5,20 +5,61 @@ phases:
  timeout: 300
  cbmcArguments:
    classpath: '/tools/cbmc/models-simple-overlay.jar:/tools/cbmc/models.jar:.'
-   max-nondet-array-length: 100
-   java-max-vla-length: false
+   java-assume-inputs-non-null: true
+   java-max-vla-length: 48
+   max-nondet-array-length: 10
+   max-nondet-string-length: 10
+   string-printable: true
+   throw-runtime-exceptions: false
    unwind: 1
+ nextPhase:
+   not_analyzed: null
+   time_out: 3
 -
  timeout: 300
  cbmcArguments:
    classpath: '/tools/cbmc/models-simple-overlay.jar:/tools/cbmc/models.jar:.'
-   max-nondet-array-length: 100
-   java-max-vla-length: false
+   java-assume-inputs-non-null: false
+   java-max-vla-length:  96
+   max-nondet-array-length: 20
+   max-nondet-string-length: 50
+   string-printable: true
+   throw-runtime-exceptions: false
    unwind: 2
+ nextPhase:
+   time_out: 3
 -
  timeout: 300
  cbmcArguments:
    classpath: '/tools/cbmc/models.jar:.'
-   max-nondet-array-length: 100
-   java-max-vla-length: false
+   java-assume-inputs-non-null: false
+   java-max-vla-length:  192
+   max-nondet-array-length: 30
+   max-nondet-string-length: 10
```

Comment:
:question: I know this is how it appears in current `diffblue.yml` files but I think this is a mistake. The value shouldn't be lower than in phase 2. Maybe set it to 100? I think it's 100 somewhere in phase 3...

Reply:
Yes, it's 100 in the platform default options. Will change it to 100.